packages/persist/lib/routes/environment/environmentId/connection/connectionId/getRecords.ts

```diff
@@ -0,0 +1,55 @@
+import type { ApiError, Endpoint, GetRecordsSuccess } from '@nangohq/types';
+import type { EndpointRequest, EndpointResponse, RouteHandler, Route } from '@nangohq/utils';
+import { records } from '@nangohq/records';
+import { validateGetRecords } from './validate.js';
+
+type GetRecords = Endpoint<{
+    Method: typeof method;
+    Path: typeof path;
+    Params: {
+        environmentId: number;
+        nangoConnectionId: number;
+    };
+    Querystring: {
+        model: string;
+        externalIds: string[];
+        cursor: string;
+    };
+    Error: ApiError<'get_records_failed'>;
+    Success: GetRecordsSuccess;
+}>;
+
+export const path = '/environment/:environmentId/connection/:nangoConnectionId/records';
+const method = 'GET';
+
+const validate = validateGetRecords<GetRecords>();
+
+const handler = async (req: EndpointRequest<GetRecords>, res: EndpointResponse<GetRecords>) => {
+    const {
+        params: { nangoConnectionId },
+        query: { model, externalIds, cursor }
+    } = req;
+    const result = await records.getRecords({
+        connectionId: nangoConnectionId,
+        model,
+        cursor,
+        externalIds,
+        limit: 100
```

Comment:
should the caller pass the limit (with a max value allowed)?

Reply:
@TBonnin I was thinking since this will be wrapped up behind a function we're exposing that we could always keep limit at 100 and then abstract it away in the sdk. What do you think?

Reply:
Fixed in ef46920b8